runconfig/parse.go

```diff
@@ -98,6 +98,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flSwappiness        = cmd.Int64([]string{"-memory-swappiness"}, -1, "Tuning container memory swappiness (0 to 100)")
 		flNetMode           = cmd.String([]string{"-net"}, "default", "Set the Network for the container")
 		flMacAddress        = cmd.String([]string{"-mac-address"}, "", "Container MAC address (e.g. 92:d0:c6:0a:29:33)")
+		flIp4Addresses      = cmd.String([]string{"-ip-address"}, "", "Container IP4 addresses separated by comma (e.g. 10.4.4.2, 10.2.2.2)")
```

Comment:
Also: It seems that the IP addresses from this parameter are still passed directly into ParseCIDR (after splitting on comma), which still requires /32. Either, update the documentation, or add the /32 in buildCreateEndpointOptions() (let me know if you do the latter, because that will require a change in MicroDeployer).